R/ancova.R

```diff
@@ -600,7 +600,14 @@ AncovaInternal <- function(jaspResults, dataset = NULL, options) {
 }
 
 .anovaContrastsTable <- function(anovaContainer, dataset, options, ready) {
-  if (!is.null(anovaContainer[["contrastContainer"]]) || all(grepl("none", options$contrasts)))
+  #contrasts are encoded so first decode that so we can later check for things like "none" and "custom"
+  decodedContrasts <- c()
+  for (c in 1:length(options$contrasts)) {
+    options$contrasts[c]$decoded <- jaspBase::decodeColNames(options$contrasts[c]$contrast)
+    append(decodedContrasts, options$contrasts[c]$decoded)
+  }
+
+  if (!is.null(anovaContainer[["contrastContainer"]]) || all(grepl("none", decodedContrasts)))
```

Comment:
your list indexing does not work here - if you replace lines 604-608 with:
```
  decodedContrasts <- list()
  for (i in 1:length(options$contrasts)) {
    options$contrasts[[i]]$decoded <- decodedContrasts[[i]] <- jaspBase::decodeColNames(options$contrasts[[i]]$contrast)
  }
```
it will work (before, decodedContrasts was always NULL)

Reply:
ah for the ancova you also need a 
  
```
  if (!ready)
    return()

```
before running that code

Reply:
So...
Im not sure where you want it...

The easiest might be to use the 
<img width="47" alt="image" src="https://user-images.githubusercontent.com/29062713/232854823-8bb1c5e6-103a-41e1-8798-15dd77303f8a.png">
button and make the commit suggestion there?